### PR TITLE
Smart flagship

### PR DESCRIPTION
**Pull request by j55blanchet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,18 @@ There are TF errors on `move_base` on `ROS Melodic` due to shifting (TF to TF2).
...
+* NLP Dependencies
+    ```
+    pip install cython
+    git clone https://github.com/epfml/sent2vec.git
+    cd sent2vec
+    pip install . 
+    (note you must install from inside cloned folder and cannot run pip install sent2vec)
 
+    Download pre-trained NLP model from here https://drive.google.com/open?id=0B6VhzidiLvjSOWdGM0tOX1lUNEk 
+    Rename file to 'model.bin' and move to '/<path-to-package>/src/models'
```
Comment (MingiJeong) [inline README.md:40]: not pkg/src/models --> pkg/models as Joe's path reading in the script

Comment (MingiJeong) [reply README.md:40]: specify path-to-pkg as just 'fleet_command'

File: src/fleet_command/Flagship.py [hunk 2]
```diff
@@ -32,19 +35,36 @@
...
+        self.command_map = {
+            'capture the enemy': self.respondto_capture_command,
+            'defend the flagship' : self.respondto_defend_command
+        }
+
+         # Setup smart command
+        if self.smart_command:
+            import sent2vec
+            self.model = sent2vec.Sent2vecModel()
+            self.model.load_model(ROOT+'/models/model.bin', inference_mode=True)
```
Comment (MingiJeong) [inline src/fleet_command/Flagship.py:66]: the reason for path in README

**Review by MingiJeong — COMMENTED** [review]
great work, only minor fix needed for README

**Review by JosephGatto — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (j55blanchet) [reply README.md]: updated to clarify

Comment (j55blanchet) [reply README.md]: (people may name the folder something else)

**Pull request merged**